utils.go

```diff
@@ -270,6 +272,7 @@ const (
 	MethodConnect = "CONNECT" // RFC 7231, 4.3.6
 	MethodOptions = "OPTIONS" // RFC 7231, 4.3.7
 	MethodTrace   = "TRACE"   // RFC 7231, 4.3.8
+	methodUse     = "USE"
```

Comment:
Maybe we should change it.
```suggestion
	MethodUse     = "USE"
```

Reply:
Now it’s just used inside the package.

Reply:
it's okay, hehehe my mistake